### PR TITLE
Feature/mds 2542 - Adding Unit Codes, updating underground activity type

### PR DESCRIPTION
**Pull request by mmumby**
Opening the initial PR:
- added unit codes, updated some positioning so the new fields fit within the form. 
- Minor styling change to the Admin view when viewing a list of mines for proponents
Screen Shot 2020-01-07 at 2 19 07 PM
- added a new field input that has a built in drop down for situations like unit codes
- Will remove the `staticContent` fetches once Ryans PR is merged.

### CODE REVIEW

File: services/core-web/src/components/admin/MinespaceUserList.js [hunk 1]
```diff
@@ -33,7 +33,13 @@ const columns = [
...
     render: (text) => (
       <div title="Mines">
-        {text && text.map(({ mine_guid, mine_name }) => <span key={mine_guid}> {mine_name} </span>)}
+        {text &&
+          text.map(({ mine_guid, mine_name }) => (
+            <span key={mine_guid}>
+              {mine_name}
+              <br />
+            </span>
+          ))}
```
Comment (exzizt) [inline services/core-web/src/components/admin/MinespaceUserList.js]: Do we want to display a message if they have no mines? Or is this dealt with already?

Comment (mmumby) [reply services/core-web/src/components/admin/MinespaceUserList.js]: chatted IRL - there should never be a situation where there isn't at least 1 mine in this list.

File: migrations/sql/V2020.01.09.9.30__update_sand_gravel_operation.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ADD COLUMN average_overburden_depth_unit_type_code character varying(3),
+ADD COLUMN average_top_soil_depth_unit_type_code character varying(3);
+
+
+ALTER TABLE sand_gravel_quarry_operation
+ADD FOREIGN KEY (average_overburden_depth_unit_type_code) REFERENCES unit_type(unit_type_code) DEFERRABLE INITIALLY DEFERRED,
+ADD FOREIGN KEY (average_top_soil_depth_unit_type_code) REFERENCES unit_type(unit_type_code) DEFERRABLE INITIALLY DEFERRED;
+
+
+UPDATE unit_type set unit = 't' where unit_type_code = 'MTN';
```
Comment (nayr974) [inline migrations/sql/V2020.01.09.9.30__update_sand_gravel_operation.sql]: Ideally I think this column is named "unit" as you've used in the response model, or "short_description" or "display_value" or something along that line as "type" has a meaning in your database that might cause confusion. I'm fine leaving this as is if time constraints prevent a change.

File: migrations/sql/V2020.01.07.01.26__update_underground_exploration.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE underground_exploration
```
[the remark this answers is not in the document]
Comment (mmumby) [reply migrations/sql/V2020.01.07.01.26__update_underground_exploration.sql]: It was on the wrong table, needed to be on the on underground_exploration_detail

[the remark this answers is not in the document]
Comment (mmumby) [reply migrations/sql/V2020.01.07.01.26__update_underground_exploration.sql]: Which it already existed on.

File: migrations/sql/V2020.01.09.9.30__update_sand_gravel_operation.sql (hunk 2 above)
Comment (mmumby) [reply migrations/sql/V2020.01.09.9.30__update_sand_gravel_operation.sql]: I can update this!

**Pull request merged**